tests/sqlparser_common.rs

```diff
@@ -4333,12 +4326,14 @@ fn parse_merge() {
     let sql = "MERGE INTO s.bar AS dest USING (SELECT * FROM s.foo) as stg ON dest.D = stg.D AND dest.E = stg.E WHEN NOT MATCHED THEN INSERT (A, B, C) VALUES (stg.A, stg.B, stg.C) WHEN MATCHED AND dest.A = 'a' THEN UPDATE SET dest.F = stg.F, dest.G = stg.G WHEN MATCHED THEN DELETE";
     match verified_stmt(sql) {
```

Comment:
Can you please also add a test for `MERGE` without the `INTO`?

Reply:
Thanks -- I don't seem to see a new commit on this PR yet. Did you push one?

Reply:
Force pushed previous one 🙂